### PR TITLE
Adding pynetdicom

### PR DESCRIPTION
**Pull request by scaramallion**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] ~If static libraries are linked in, the license of the static library is packaged.~ N/A
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] ~GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there~ N/A

### CODE REVIEW

**Comment by scaramallion** [comment]
@conda-forge-admin, please rerender

**Comment by scaramallion** [comment]
@conda-forge-admin, please rerender

**Comment by scaramallion** [comment]
@conda-forge-admin, please restart ci

**Comment by scaramallion** [comment]
@conda-forge-admin, please rerender

**Comment by scaramallion** [comment]
@conda-forge-admin please rerender

**Comment by scaramallion** [comment]
@conda-forge-admin, please re-render

**Comment by scaramallion** [comment]
@conda-forge-admin, please rerender

File: recipes/pynetdicom/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+about:
+  home: https://github.com/pydicom/pynetdicom
+  license: MIT
+  license_family: MIT
+  license_file: LICENCE.txt
+  summary: 'A Python implementation of the DICOM networking protocol'
+
+  # The remaining entries in this section are optional, but recommended.
+  description: |
+    `DICOM <http://dicom.nema.org>`_ is the international standard for medical
```
Comment (scopatz) [inline recipes/pynetdicom/meta.yaml]: Please remove rST formatting

File: recipes/pynetdicom/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/pydicom/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 1
```
Comment (scopatz) [inline recipes/pynetdicom/meta.yaml]: The build number should be zero

File: recipes/pynetdicom/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - python
+    - pip
+    - setuptools
```
Comment (scopatz) [inline recipes/pynetdicom/meta.yaml]: please remove setuptools

File: recipes/pynetdicom/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  home: https://github.com/pydicom/pynetdicom
+  license: MIT
+  license_family: MIT
+  license_file: LICENCE.txt
+  summary: 'A Python implementation of the DICOM networking protocol'
+  description: |
+    *pynetdicom* is a pure Python package that implements the
+    DICOM <http://dicom.nema.org> networking protocol. Working with pydicom
+    <https://github.com/pydicom/pydicom>, it allows the easy creation of
+    DICOM *Service Class Users* (SCUs) and *Service Class Providers* (SCPs)
```
Comment (scopatz) [inline recipes/pynetdicom/meta.yaml]: please remove markup from text

**Comment by scaramallion** [comment]
Thanks! Sorry for the trouble with the description.

**Comment by scopatz** [comment]
np!

**Pull request merged**